src/UIImage/UIImage+ColorMaskedImage.h

```diff
@@ -0,0 +1,22 @@
+//
+//  UIImage+ColorMaskedImage.h
+//  Chronability
+//
+//  Created by Logan Wright on 3/18/15.
+//  Copyright (c) 2015 IntrepidPursuits. All rights reserved.
+//
```

Comment:
I have been removing the header / copyright information.  Do we want to continue with keeping them or removing them?

@brightredchilli @LoganWright ?